src/components/Menu/Menu.tsx

```diff
@@ -62,6 +62,7 @@ const Menu: React.FC<Props & TestProps & EventProps> = props => {
     leftIconName,
     iconSize = 16,
     avatar,
+    dataTestId,
```

Comment:
Optional: maybe this testID should refer to the wrapper <div> and to the open/close button.